frontend/controller/actions/group.js

```diff
@@ -44,6 +43,15 @@ export default sbp('sbp/selectors/register', {
 
     try {
       const initialInvite = createInvite({ quantity: 60, creator: INVITE_INITIAL_CREATOR })
+      const proposalSettings = {
+        rule: ruleName,
+        ruleSettings: {
+          [ruleName]: {
+            ...thresholdAdjusted(ruleName, +ruleThreshold, 1), // 1 = group initial size.
```

Comment:
Threshold should not be adjusted here or with a `thresholdAdjusted` function, see comment below for details.